assets/in

```diff
@@ -76,6 +76,7 @@ if [ "$skip_download" = "false" ]; then
 
   echo "$image_id" > ${destination}/image-id
   docker inspect $image_id > ${destination}/docker_inspect.json
+  labels="$(cat ${destination}/docker_inspect.json | jq -r '.[0].Config.Labels // {} | to_entries | map("\(.key)=\(.value)") | join(", ")')"
```

Comment:
what happens if the jq parsing encounters problem ? would it be always make labels empty only?

Could you add a test to make sure all cases are considered so it wont errored out?